sourcemash/api/items.py

```diff
@@ -17,18 +16,46 @@
     'category_1': fields.String,
     'category_2': fields.String,
     'summary': fields.String,
-    'category_1': fields.String,
-    'category_2': fields.String,
+    'totalVotes': fields.Integer,
     'uri': fields.Url('api.item')
 }
 
 
 class ItemAPI(Resource):
+    def __init__(self):
+        self.reqparse = reqparse.RequestParser()
+        self.reqparse.add_argument('vote', type = int, required = True)
+        super(ItemAPI, self).__init__()
 
     def get(self, id):
         item = Item.query.get_or_404(id)
         return {'item': marshal(item, item_fields)}
 
+    @login_required
+    def put(self, id):
+        ''' Update item vote count '''
+        args = self.reqparse.parse_args()
+        item = Item.query.get_or_404(id)
+        
+        # Reject if size of vote is too large
+        if abs(args.vote) > 1:
+            return {'item': marshal(item, item_fields)}, 422 
```

Comment:
This should return a dictionary of errors like below, but that will be covered once we write a form to validate the request.